x/application/client/cli/tx_stake_application_test.go

```diff
@@ -38,96 +39,114 @@ func TestCLI_StakeApplication(t *testing.T) {
 		fmt.Sprintf("--%s=%s", flags.FlagFees, sdk.NewCoins(sdk.NewCoin(net.Config.BondDenom, sdkmath.NewInt(10))).String()),
 	}
 
+	defaultConfig := `
+		service_ids:
+		  - svc1
+		  - svc2
+		  - svc3
+		`
+
 	tests := []struct {
-		desc             string
-		address          string
-		stakeString      string
-		serviceIdsString string
-		err              *sdkerrors.Error
+		desc        string
+		address     string
+		stakeString string
+		config      string
+		err         *sdkerrors.Error
 	}{
 		// Happy Paths
 		{
-			desc:             "valid",
-			address:          appAccount.Address.String(),
-			stakeString:      "1000upokt",
-			serviceIdsString: "svc1,svc2,svc3",
-			err:              nil,
+			desc:        "valid",
+			address:     appAccount.Address.String(),
+			stakeString: "1000upokt",
+			config:      defaultConfig,
+			err:         nil,
 		},
 
 		// Error Paths - Address Related
 		{
 			desc: "address_test: missing address",
 			// address:     "explicitly missing",
-			stakeString:      "1000upokt",
-			serviceIdsString: "svc1,svc2,svc3",
-			err:              types.ErrAppInvalidAddress,
+			stakeString: "1000upokt",
+			config:      defaultConfig,
+			err:         types.ErrAppInvalidAddress,
 		},
 		{
-			desc:             "stake application: invalid address",
-			address:          "invalid",
-			stakeString:      "1000upokt",
-			serviceIdsString: "svc1,svc2,svc3",
-			err:              types.ErrAppInvalidAddress,
+			desc:        "stake application: invalid address",
+			address:     "invalid",
+			stakeString: "1000upokt",
+			config:      defaultConfig,
+			err:         types.ErrAppInvalidAddress,
 		},
 
 		// Error Paths - Stake Related
 		{
 			desc:    "address_test: missing stake",
 			address: appAccount.Address.String(),
 			// stakeString: "explicitly missing",
-			serviceIdsString: "svc1,svc2,svc3",
-			err:              types.ErrAppInvalidStake,
+			config: defaultConfig,
+			err:    types.ErrAppInvalidStake,
 		},
 		{
-			desc:             "address_test: invalid stake denom",
-			address:          appAccount.Address.String(),
-			stakeString:      "1000invalid",
-			serviceIdsString: "svc1,svc2,svc3",
-			err:              types.ErrAppInvalidStake,
+			desc:        "address_test: invalid stake denom",
+			address:     appAccount.Address.String(),
+			stakeString: "1000invalid",
+			config:      defaultConfig,
+			err:         types.ErrAppInvalidStake,
 		},
 		{
-			desc:             "address_test: invalid stake amount (zero)",
-			address:          appAccount.Address.String(),
-			stakeString:      "0upokt",
-			serviceIdsString: "svc1,svc2,svc3",
-			err:              types.ErrAppInvalidStake,
+			desc:        "address_test: invalid stake amount (zero)",
```

Comment:
I'm a bit confused because:

1. The test is called `TestCLI_StakeApplication`
2. The prefix for the tests is `address_test`

I think:
1. The prefix should be `invalid` or `valid`
2. We don't need `address_test` or `stake_application` anywhere here.

IMPORTANT: I believe this was like this BEFORE your changes so I apologize for that, but would appreciate if you can update it!